components/raft_log_engine/Cargo.toml

```diff
@@ -14,7 +14,7 @@ num_cpus = "1"
 online_config = { path = "../online_config" }
 protobuf = "2"
 raft = { version = "0.7.0", default-features = false, features = ["protobuf-codec"] }
-raft-engine = { git = "https://github.com/tikv/raft-engine.git", features = ["swap"] }
+raft-engine = { git = "https://github.com/jiayang-zheng/raft-engine.git", rev = "9200bdae002a2fa09a56e0753eae211477f03f48", features = ["swap"] }
```

Comment:
Same as above.

Reply:
This is WIP, I will change it back after raft-engine repo merge related pr.